telnyx_rtc/src/main/java/com/telnyx/webrtc/sdk/TelnyxClient.kt

```diff
@@ -1274,6 +1279,7 @@ class TelnyxClient(
                     peerConnection?.onRemoteSessionReceived(sdp)
 
                     callStateLiveData.postValue(CallState.ACTIVE)
+                    callStateLiveData.value = CallState.ACTIVE
```

Comment:
Same as before 
In line `1281` we are using postValue, which is thread-safe, and in the next line, we are using .value, which is not thread-safe, and they perform almost the same operation.